streams.groovy

```diff
@@ -8,6 +8,9 @@ production = ['testing', 'stable', 'next']
 development = ['testing-devel'] + next_devel
 mechanical = ['rawhide', 'branched' /* 'bodhi-updates', 'bodhi-updates-testing' */]
 
+// list of secondary architectures we support
+secondary_arches = ['aarch64']
```

Comment:
any ideas for another term than "secondary". I think that's a loaded term in Fedora from the time where secondary arches were built in an entirely different build env etc..

`alternative_arches` maybe?

Reply:
Went with `additional_arches`!